{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "aaf87545_3868e03e",
        "filename": "/COMMIT_MSG",
        "patchSetId": 19
      },
      "lineNbr": 101,
      "author": {
        "id": 1000020
      },
      "writtenOn": "2023-10-02T21:15:45Z",
      "side": 1,
      "message": "what is this internal error from?",
      "range": {
        "startLine": 88,
        "startChar": 0,
        "endLine": 101,
        "endChar": 3
      },
      "revId": "3d06cc5ca581d7e6bae08a18a4e5e98277b5f879",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}